lib/Pod/Simple/XHTML.pm

```diff
@@ -276,7 +276,7 @@ sub new {
   my $new = $self->SUPER::new(@_);
   $new->{'output_fh'} ||= *STDOUT{IO};
   $new->perldoc_url_prefix('https://metacpan.org/pod/');
-  $new->man_url_prefix('http://man.he.net/man');
+  $new->man_url_prefix('https://man7.org/linux/man-pages');
```

Comment:
This generates invalid links.

man7.org needs both a prefix and a suffix, and the suffix is not a fixed string (it includes the section number), which the existing code structure does not handle.

Reply:
I think currently the overridable `resolve_man_page_link()` method might be the best place to put such dynamic "midfixes" 🙂

And it already contains a reference to `$section`.